src/content/it/2020/markup.md

```diff
@@ -0,0 +1,1340 @@
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Elemento</th>
+        <th>Probabilità</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>title</code></td>
+        <td class="numeric">99.34%</td>
+      </tr>
+      <tr>
+        <td><code>meta</code></td>
+        <td class="numeric">99.00%</td>
+      </tr>
+      <tr>
+        <td><code>div</code></td>
+        <td class="numeric">98.42%</td>
+      </tr>
+      <tr>
+        <td><code>a</code></td>
+        <td class="numeric">98.32%</td>
+      </tr>
+      <tr>
+        <td><code>link</code></td>
+        <td class="numeric">97.79%</td>
+      </tr>
+      <tr>
+        <td><code>script</code></td>
+        <td class="numeric">97.73%</td>
+      </tr>
+      <tr>
+        <td><code>img</code></td>
+        <td class="numeric">95.83%</td>
+      </tr>
+      <tr>
+        <td><code>span</code></td>
+        <td class="numeric">93.98%</td>
+      </tr>
+      <tr>
+        <td><code>p</code></td>
+        <td class="numeric">88.71%</td>
+      </tr>
+      <tr>
+        <td><code>ul</code></td>
+        <td class="numeric">87.68%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Elevate probabilità di trovare un dato elemento nelle pagine del campione Web Almanac 2020.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
+</figure>
+
+Gli elementi standard sono quelli che fanno o facevano parte della specifica HTML. Quali sono rari da trovare? Nel nostro esempio, ciò porterebbe a quanto segue:
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Elemento</th>
+        <th>Probabilità</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>dir</code></td>
+        <td class="numeric">0.0082%</td>
+      </tr>
+      <tr>
+        <td><code>rp</code></td>
+        <td class="numeric">0.0087%</td>
+      </tr>
+      <tr>
+        <td><code>basefont</code></td>
+        <td class="numeric">0.0092%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Scarse probabilità di trovare un dato elemento nelle pagine del campione.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
+</figure>
+
+Includiamo questi elementi per dare un'idea di quali elementi potrebbero perdere di popolarità. Ma mentre `dir` e `basefont` sono stati specificati l'ultima volta in XHTML 1.0 (2000) e non fanno più parte dell'HTML, il raro uso di `rp` (che è stato menzionato [già nel 1998](https://www.w3.org/TR/1998/WD-ruby-19981221/#a2-4) ed è [ancora parte dell'HTML](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element)), potrebbe semplicemente suggerire che il <a hreflang="en" href="https://www.w3.org/TR/ruby/">markup di Ruby</a> non è molto popolare.
+
```

Comment:
Two lines here to match English. It really should only be one line so English is wrong! If you're happy to fix English version instead, then that's the better fix.

```suggestion


```

Reply:
Fixed this and added to this branch.